static/assets/contribute-templates/how-to-template.md

```diff
@@ -1,36 +1,624 @@
-# Guide title [answers "How do I ...?]
+# My block length is too small to do forkless runtime upgrades, what do I do?
 
-Replace this paragraph with an overview that provides a brief summary of what this guide is about and why it's useful.
-The overview section can be more than one paragraph. It does not require any heading before the text.
+As you may be aware at this point, runtime upgrades must always abide by the block time requirements -- regardless of running it as SUDO or not. For example, if your block length is 100KB and your runtime WASM is 120KB, then you will not be able to perform a chain upgrade as it is too large. 
```

Comment:
I can easily think of tweaking `frame_system` and its signed extensions to allow only certain transactions to go over the block length limit. This will ensure that something like `set_code` will never exhaust the block resources. 

There should be warnings that doing this kind of thing is a bit risky.

Reply:
This kind of tweak would be **tremendously** helpful, as we were surprised that we could not override the block length limit using Sudo.

I agree with you about warnings, added them in to the article..